### DISCUSSION TITLE
compile milvus is failed

### DISCUSSION THREAD

**Discussion by weekstudy** [milvus-io/milvus#15730] [Q&A and General discussion]
![image](https://user-images.githubusercontent.com/39413303/155466220-83b467f9-9adb-48c7-b547-86563aee244c.png)
I compile the source code by the steps,but is failed,the error as the picture show . How solve it ?

**Comment by yhmo** [comment]
Are you compiling the v1.x?
Workaround:

1. import the library to [Gitee](https://gitee.com/), you can refer this article: https://blog.csdn.net/luobing4365/article/details/105658274
2. replace the download url in the milvus/core/src/index/cmake/ThirdPartyPackagesCore.cmake, line No.226
```
set(OPENBLAS_SOURCE_URL
            "https://github.com/xianyi/OpenBLAS/archive/v${OPENBLAS_VERSION}.tar.gz")
```
change the url to gitee address that you import in the step 1

3. compile again

**Reply by weekstudy** [reply]
No, compiling the V2.0

**Reply by weekstudy** [reply]
> Are you compiling the v1.x? Workaround:
> 
> 1. import the library to [Gitee](https://gitee.com/), you can refer this article: https://blog.csdn.net/luobing4365/article/details/105658274
> 2. replace the download url in the milvus/core/src/index/cmake/ThirdPartyPackagesCore.cmake, line No.226
> 
> ```
> set(OPENBLAS_SOURCE_URL
>             "https://github.com/xianyi/OpenBLAS/archive/v${OPENBLAS_VERSION}.tar.gz")
> ```
> 
> change the url to gitee address that you import in the step 1
> 
> 3. compile again

I try

**Reply by xiaofan-luan** [reply]
what is the platform you are using?

**Reply by weekstudy** [reply]
> what is the platform you are using?
Ubuntu18.04
